src/Symfony/Component/Config/Definition/Builder/NodeDefinition.php

```diff
@@ -356,4 +358,26 @@ protected function normalization()
      * @throws InvalidDefinitionException When the definition is invalid
      */
     abstract protected function createNode();
+
+    /**
+     * Set PathSeparator to use.
+     *
+     * @param string $separator
+     *
+     * @return $this
+     */
+    public function setPathSeparator($separator)
+    {
+        if ($this instanceof ChildrenAwareNodeDefinitionInterface) {
```

Comment:
`if ($this instanceof ArrayNodeDefinition) {` would be enough for the prupose, isn't it?
about the method, my preference is `getChildNodeDefinitions`